CDP4IMEInstaller/CDP4Scripting.wxs

```diff
@@ -30,7 +30,19 @@
       <Component Id="comp_MicrosoftScriptingMetadata" Guid="{F8FEBC24-58F4-406F-B1E8-2DA0A0D9EF5A}" Location="local" Win64="$(var.Win64)">
         <File Id="fil_MicrosoftScriptingMetadata"  Vital="yes" Source="$(var.CDP4Scripting.TargetDir)Microsoft.Scripting.Metadata.dll" KeyPath="yes"></File>
       </Component>
-      
+

```

Comment:
i wonder if these references are not better suited in the generic dependencies

Reply:
Added ticket #1250